src/main/java/org/peergos/util/Config.java

```diff
@@ -0,0 +1,60 @@
+package org.peergos.util;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Config {
+    private final Map configuration;
```

Comment:
Can we enforce that this is a Map<String, Object> or better?

Reply:
done